lib/stack-functions

```diff
@@ -270,8 +270,7 @@ stack-parameters() {
   aws cloudformation describe-stacks        \
     --stack-name ${stack}                   \
     --query "${query}"                      \
-    --output ${output:-json}                |
-    jq --sort-keys .
+    --output ${output:-json}
```

Comment:
Sorting the keys in this instance results in ParameterKey preceding ParameterValue.

This is an (almost) self contained function. I would prefer to test for $output rather than default to ugly JSON output